js/foundation.util.touch.js

```diff
@@ -140,15 +140,10 @@ class Touch {}
   };
 
   Touch.init = function($) {
```

Comment:
*Although*, if you go down this road, consider which libraries would be included in a bundle, and which ones would still need to be included separately. For example, jQuery is not bundled in `foundation.min.js`, as its assumed you've loaded it in a separate `<script>` tag.

If you swap out some internals for things like Hammer, it's probably fine to bundle it in `foundation.min.js`. Otherwise, any time you wanted to add a new dependency, non-build-system users would need to add new `<script>` tags to load the new dependencies.

Of course, this makes it possible to have duplicate copies of libraries in your code. Like maybe you're using Foundation *and* Hammer directly, and now you have two copies of Hammer in your code. However, I'd consider this a moot point because if you really care about that stuff, you're probably using a build system.